apps/filebrowser/src/filebrowser/api.py

```diff
@@ -496,18 +496,31 @@ def rename(request):
   source_path = request.POST.get('source_path', '')
   destination_path = request.POST.get('destination_path', '')
 
+  # Check if source and destination paths are provided
+  if not source_path or not destination_path:
```

Comment:
When will this scenario occur?

Reply:
If someone doesn't call the API correctly, I've added unit test for this scenario.